lib/meilisearch-rails.rb

```diff
@@ -293,9 +293,10 @@ def additional_indexes
   # this class wraps an Algolia::Index object ensuring all raised exceptions
   # are correctly logged or thrown depending on the `raise_on_failure` option
   class SafeIndex
-    def initialize(index_uid, raise_on_failure)
+    def initialize(index_uid, raise_on_failure, options)
       client = MeiliSearch.client
-      @index = client.get_or_create_index(index_uid)
+      options[:primary_key].nil? ?  primary_key = {} : primary_key = { primaryKey: options[:primary_key].to_s }
+      @index = client.get_or_create_index(index_uid, primary_key)
```

Comment:
```suggestion
      primary_key = options[:primary_key].to_s ||  MeiliSearch::IndexSettings::DEFAULT_PRIMARY_KEY
      @index = client.get_or_create_index(index_uid, { primaryKey: primary_key })
```

Reply:
You should add `DEFAULT_PRIMARY_KEY` in `MeiliSearch::IndexSettings` of course 😇